doc/api/readline.md

```diff
@@ -507,21 +532,25 @@ changes:
   * `terminal` {boolean} `true` if the `input` and `output` streams should be
     treated like a TTY, and have ANSI/VT100 escape codes written to it.
     **Default:** checking `isTTY` on the `output` stream upon instantiation.
+  * `history` {array} Initial list of history lines. This option makes sense
```

Comment:
We prefer not to use `array` in our documentation, it's more useful to document what's inside the array.
```suggestion
  * `history` {string[]} Initial list of history lines. This option makes sense
```

Reply:
Done